com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/TextureSetLayout.java

```diff
@@ -588,11 +592,11 @@ public static List<Layout> createTextureSet(List<Page> pages) {
                 rect.setPage(page.index);
                 rect.setRotated(image.rotated);
 
-                Pointi p;
+                Point p;
                 if (image.pivot != null)
-                    p = new Pointi(image.pivot);
+                    p = new Point(image.pivot);
```

Comment:
If image size was 1x1, and the pivot 0.5,0.5, then pivot would be clamped to 0,0.
Which is quite a shift for a 1x1 size image.